packages/server/tests/integration/server.spec.ts

```diff
@@ -485,6 +485,81 @@ describe('RPC Server', function () {
     expect(res.data.result).to.be.equal('0x0');
   });
 
+  it('should execute "engine_getPayloadV1" and return UNSUPPORTED_METHOD', async function () {
```

Comment:
We can remove duplications by creating a list of methods, iterating them, and creating an `it` statement for each, similar to [here](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/tests/lib/mirrorNodeClient.spec.ts#L60).